ocdata/analysis.py

```diff
@@ -0,0 +1,66 @@
+"""
+This submodule contains the scripts for VASP calculations
+post-analysis:
+
+(1) Bader Charges Analysis
+(2) pDOS Analysis (TODO)
+"""
+
+__author__ = 'Javier Heras-Domingo'
+__email__ = 'jherasdo@andrew.cmu.edu'
+
+import os
+import sys
+import pickle
+from shutil import which
+
+import warnings
+warnings.filterwarnings("ignore")
+
+
+
+#Run Bader Analysis after vasp calculation
```

Comment:
Please add single spaces after `#`. Comments are more difficult to read without this space, as the `#` blurs together with the first character.

Reply:
Also:  This comment should really be a part of the docstring for the function, above the `Arguments` section. This way, users that type `help(run_bader)` get to see the description, not just the people looking at the code.